dev-workflow-ui/src/ch/ivyteam/workflowui/notification/NotificationBean.java

```diff
@@ -0,0 +1,60 @@
+package ch.ivyteam.workflowui.notification;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.faces.bean.ManagedBean;
+import javax.faces.bean.ViewScoped;
+
+import ch.ivyteam.ivy.notification.web.WebNotifications;
+import ch.ivyteam.ivy.security.ISession;
+
+@ViewScoped
+@ManagedBean
+public class NotificationBean {
+
+  private final WebNotifications webNotifications;
+
+  private long countAll;
+  private long countUnread;
+
+  public NotificationBean() {
+    this.webNotifications = WebNotifications.of(ISession.current().getSessionUser());
+    this.countAll = webNotifications.countAll();
+    this.countUnread = webNotifications.countUnread();
+  }
+
+  public List<NotificationDto> getNotifications() {
+    return webNotifications.read(0, 100).stream()
+            .map(NotificationDto::new)
+            .collect(Collectors.toList());
+  }
+
+  public void deleteAll() {
+    webNotifications.deleteAll();
```

Comment:
Instead of deleting it mark it as hide. Then the current user won't see it anymore but admins still can investigate problems

Reply:
Would mean a new column in IWA_Notification. We can do that later.